### PR TITLE
[CARBONDATA-3791] Index server doc changes

### PR DESCRIPTION
**Pull request by vikramahuja1001**
### Why is this PR needed?
 Changing index server docs as per the code.
 
 ### What changes were proposed in this PR?
Changing index server docs as per the code.
    
 ### Does this PR introduce any user interface change?
 - No

 ### Is any new testcase added?
 - No

### CODE REVIEW

File: docs/index-server.md [hunk 1]
```diff
@@ -21,7 +21,7 @@
 
 Carbon currently prunes and caches all block/blocklet datamap index information into the driver for
 normal table, for Bloom/Index datamaps the JDBC driver will launch a job to prune and cache the
-datamaps in executors.
+indexes in executors.
```
Comment (Indhumathi27) [inline docs/index-server.md:24]: Remove datamap keyword in Line No.22, Line No.23, Line no.105, Line no.123, Line No.155  and Line no.159

**Comment by CarbonDataQA1** [comment]
Build Success with Spark 2.3.4, Please check CI http://121.244.95.60:12545/job/ApacheCarbonPRBuilder2.3/2923/

File: docs/index-server.md (hunk 1 above)
Comment (vikramahuja1001) [reply docs/index-server.md:24]: that is done in PR3738

**Comment by CarbonDataQA1** [comment]
Build Success with Spark 2.4.5, Please check CI http://121.244.95.60:12545/job/ApacheCarbon_PR_Builder_2.4.5/1206/

File: docs/index-server.md (hunk 1 above)
Comment (Indhumathi27) [reply docs/index-server.md:24]: ok

Comment (kunal642) [reply docs/index-server.md:24]: handled in 3738

File: docs/index-server.md [hunk 2]
```diff
@@ -21,7 +21,7 @@
 
 Carbon currently prunes and caches all block/blocklet datamap index information into the driver for
```
Comment (chetandb) [inline docs/index-server.md:22]: Remove datamap keyword whereever mentioned

Comment (vikramahuja1001) [reply docs/index-server.md:22]: These changes are handled in PR3738

File: docs/index-server.md [hunk 3]
```diff
@@ -152,11 +152,11 @@ The user can enable prepriming by using 'carbon.indexserver.enable.prepriming' =
...
 | carbon.index.server.port | NA | The port on which the index server has to be started. |
 |carbon.index.server.max.worker.threads| 500 | Number of RPC handlers to open for accepting the requests from JDBC driver. Max accepted value is Integer.Max. Refer: [Hive configuration](https://github.com/apache/hive/blob/master/common/src/java/org/apache/hadoop/hive/conf/HiveConf.java#L3441) |
-|carbon.max.executor.lru.cache.size|  NA | Maximum memory **(in MB)** upto which the executor process can cache the data (DataMaps and reverse dictionary values). Only integer values greater than 0 are accepted. **NOTE:** Mandatory for the user to set. |
-|carbon.index.server.max.jobname.length|NA|The max length of the job to show in the index server application UI. For bigger queries this may impact performance as the whole string would be sent from JDBCServer to IndexServer.|
+|carbon.max.executor.lru.cache.size|  -1 | Maximum memory **(in MB)** upto which the executor process can cache the data (DataMaps and reverse dictionary values). Only integer values greater than 0 are accepted. **NOTE:** Mandatory for the user to set. |
```
Comment (kunal642) [inline docs/index-server.md:155]: For index server default value is not set..We are throwing exception if user has not set this

Comment (vikramahuja1001) [reply docs/index-server.md]: reverted it back

**Comment by CarbonDataQA1** [comment]
Build Success with Spark 2.4.5, Please check CI http://121.244.95.60:12545/job/ApacheCarbon_PR_Builder_2.4.5/1242/

**Comment by CarbonDataQA1** [comment]
Build Success with Spark 2.3.4, Please check CI http://121.244.95.60:12545/job/ApacheCarbonPRBuilder2.3/2960/

**Comment by kunal642** [comment]
Please rebase

**Comment by vikramahuja1001** [comment]
rebase completed

**Comment by CarbonDataQA1** [comment]
Build Success with Spark 2.3.4, Please check CI http://121.244.95.60:12545/job/ApacheCarbonPRBuilder2.3/2968/

**Comment by CarbonDataQA1** [comment]
Build Success with Spark 2.4.5, Please check CI http://121.244.95.60:12545/job/ApacheCarbon_PR_Builder_2.4.5/1250/

**Comment by kunal642** [comment]
LGTM

**Pull request closed without merging**